docs/src/components/reference/AttributesTable.tsx

```diff
@@ -38,10 +38,12 @@ export async function AttributesTable({ data, ...props }: AttributesTableProps)
             useMDXComponents: () => inlineMdxComponents,
           });
 
+          const attributeName = name.startsWith('[') ? name : `[${name}]`;
```

Comment:
Do we need to show brackets in the docs? Since they will be around every attribute, it's more noise IMO.

Reply:
I think they mainly help you identify that you are looking at attributes for styling, check it in the context:

https://master--base-ui.netlify.app/react/components/dialog

Reply:
I was hesitant about this too, I had to remove it from the enum so we can use it in the code as an actual value. I would personally remove, but I didn't wanted to change previous behavior.

Reply:
Cool agreed